src/d3a/models/area/__init__.py

```diff
@@ -411,10 +411,19 @@ def tick(self):
                 self.dispatcher.publish_market_clearing()
             else:
                 self._update_myco_matcher()
+                bid_offer_matcher.event_tick(
+                    **self._progress_info, current_tick_in_slot=self.current_tick_in_slot)
```

Comment:
Does it really need to be moved here ? The way it is at the moment, the event_tick method will be called on every tick for every market area in the simulation. The MycoMatcher implementation though assumes that it is called once per tick for all markets AFAIR. I think that leaving this part of the code in simulation.py as it originally was would be fine. 

Similar comment exists for the sleep method, this should also not be required.